src/sestest/GenerateCommandLine.cs

```diff
@@ -34,5 +34,8 @@ public sealed class GenerateCommandLine : CommandLineBase
 
         [Option("audience", HelpText = "Audience to which tokens will be addressed (optional; defaults to 'https://batch.azure.test/software-entitlement').")]
         public string Audience { get; set; }
+
+        [Option("issuer", HelpText = "Issuer by whom the tokens are created (optional; defaults to 'http://batch.azure.test/software-entitlement').")]
```

Comment:
Updated the docs to clarify that these are for use by Azure Batch and marked as `[Internal]` here.